### ISSUE TITLE
[RFE] Update EULA & T&C  in Rancher Helm Chart README

### ISSUE THREAD

**Issue by jiaqiluo** [rancher/rancher#39236]
The following sentence needs to be added to the README in the Rancher Helm Chart 
```
By installing this application, you accept the [End User License Agreement & Terms & Conditions](https://www.suse.com/licensing/eula/)
```

SURE-5322

**Comment by jiaqiluo** [comment]
This is an enhancement to the Helm chart. 

 
### What was fixed, or what changes have occurred

adds EULA & T&C to Rancher Helm Chart README.
 
### Areas or cases that should be tested

The sentence mentioned in the issue should show in the README.

  
### What areas could experience regressions?
 
n/a

### Are the repro steps accurate/minimal?

yes

**Comment by jameson-mcghee** [comment]
QA TEST PLAN
| Scenarios | Test Case |
|-------------------------------|
| 1 | Verify the [README file](https://github.com/rancher/charts/blob/dev-v2.7/README.md) contains the correct information in Github |
| 2 | Update local _rancher-charts_ helm chart (```helm repo update```), and check the README contains the correct information using the  ```helm show readme [CHART] [flags]``` command (https://helm.sh/docs/helm/helm_show_readme/) |

**Comment by deniseschannon** [comment]
This is available with v2.7.0-rc4

**Comment by jameson-mcghee** [comment]
I verified that the statement was added in the v2.6 Chart README file. I also verified that the URL link was to the correct address. Therefore I am closing this ticket as Done.
![image.png](https://zube.io/files/rancher/2cb3c2c8a0fa1432a1a3fc13bf3ffae1-image.png)
![image.png](https://zube.io/files/rancher/dcce9b47db988ce1dfc30cd86a8af41a-image.png)
